src/sentry/static/sentry/app/views/settings/account/accountAuthorizations.jsx

```diff
@@ -73,30 +64,29 @@ class AccountAuthorizations extends AsyncView {
       state => ({
         data: state.data.filter(({id}) => id !== authorization.id),
       }),
-      () => {
-        const loadingIndicator = IndicatorStore.add(t('Saving changes..'));
-        this.api.request('/api-authorizations/', {
-          method: 'DELETE',
-          data: {authorization: authorization.id},
-          success: data => {
-            IndicatorStore.remove(loadingIndicator);
-          },
-          error: () => {
-            this.setState({
-              data: oldData,
-            });
-            IndicatorStore.remove(loadingIndicator);
-            IndicatorStore.addError(t('Unable to save changes. Please try again.'));
-          },
-        });
+      async () => {
+        addLoadingMessage();
+
+        try {
+          await this.api.requestPromise('/api-authorizations/', {
+            method: 'DELETE',
+            data: {authorization: authorization.id},
+          });
+
+          clearIndicators();
```

Comment:
Would it be better to use `addSuccessMessage` to give feedback that the delete was completed?